### ISSUE TITLE
[BUG] - Roundcube cannot sent mail and postix,rspamd showing startup prove failed

### ISSUE THREAD

**Issue by shea786** [Mailu/helm-charts#181]
**Describe the bug**
I am setting up Mailu with Kubernetes setup
Two pods are throwing errors, and I am not sure why that is the case
- postfix
- rspamd

Both of these are showing error as "Startup probe failed" with rspamd showing that it cannot connect to itself on port 11334

When I try to send a message through roundcube, I get an error stating
Connection error (failed to reach the server) 

Apologies in advance if I have missed anything and your help us highly appreciated.

**Environment**
 - Kubernetes Platform

**Additional context**
![image](https://user-images.githubusercontent.com/17195904/174459011-b6e13e25-193a-4371-b7cc-cf6560410508.png)
![image](https://user-images.githubusercontent.com/17195904/174459025-3a57f48f-6cdf-4265-8256-ce03e612b4b7.png)

**Comment by jamesarbrown** [comment]
Regarding roundcube this is possibly #171

**Comment by gnuion** [comment]
> **Describe the bug** I am setting up Mailu with Kubernetes setup Two pods are throwing errors, and I am not sure why that is the case
> 
> * postfix
> * rspamd
> 
> Both of these are showing error as "Startup probe failed" with rspamd showing that it cannot connect to itself on port 11334
> 
> When I try to send a message through roundcube, I get an error stating Connection error (failed to reach the server)
> 
> Apologies in advance if I have missed anything and your help us highly appreciated.
> 
> **Environment**
> 
> * Kubernetes Platform
> 
> **Additional context** ![image](https://user-images.githubusercontent.com/17195904/174459011-b6e13e25-193a-4371-b7cc-cf6560410508.png) ![image](https://user-images.githubusercontent.com/17195904/174459025-3a57f48f-6cdf-4265-8256-ce03e612b4b7.png)

damn the default values.yaml is garbage. use:

```yaml
rspamd:
  resources:
    limits:
      cpu: 600m
      memory: 500Mi
    requests:
      cpu: 500m
      memory: 500Mi
```

**Comment by gnuion** [comment]
https://github.com/Mailu/helm-charts/pull/207

**Comment by fastlorenzo** [comment]
Fixed in release 1.0.0
